### DISCUSSION TITLE
MiniCPM-V和OmniLMM有关系吗

### DISCUSSION THREAD

**Discussion by sunshineflg** [OpenBMB/MiniCPM#72] [Q&A] [answered]
MiniCPM-V和OmniLMM有什么区别

**Comment by iceflame89** [comment, marked as answer]
OmniLMM包含12B和3B两个模型版本，其中OmniLMM-3B是基于MiniCPM作为LLM部分，所以又称为MiniCPM-V。两个版本多模态架构和训练方法都是一样的
